lib/impure/db_mysql.nim

```diff
@@ -359,6 +361,16 @@ proc insertId*(db: DbConn, query: SqlQuery,
   result = tryInsertID(db, query, args)
   if result < 0: dbError(db)
 
+proc tryInsert*(db: DbConn, query: SqlQuery, pkName: string,
+                  args: varargs[string, `$`]): int64
+                  {.tags: [WriteDbEffect], raises: [], since:(1, 3).} =
```

Comment:
```suggestion
proc tryInsert*(db: DbConn, query: SqlQuery, pkName: string,
                args: varargs[string, `$`]): int64
               {.tags: [WriteDbEffect], raises: [], since:(1, 3).} =
```

Please align the parameters as specified in NEP-1: https://nim-lang.org/docs/nep1.html#introduction-conventions-for-multiminusline-statements-and-expressions

Also, can you add documentation stating that this proc is the same as `tryInsertID` for this module?

Reply:
sure